src/test/java/com/appland/appmap/test/util/MethodBuilderTest.java

```diff
@@ -32,7 +32,7 @@ public void testBuild() throws Exception {
     Method myMethod = testClass.getMethod("myMethod");
 
     myMethod.invoke(obj);
-    assertEquals(myMethodMessage + "\n", systemOutRule.getLog());
+    assertEquals(myMethodMessage + "\r\n", systemOutRule.getLog());
```

Comment:
@dustinbyrne this actually fixed the issue with the failing test on windows is supposed to work in Linux too, could you check I had a bad time trying to test it using online terminals and downloading a Linux virtual machine.